### ISSUE TITLE
Unable to download reference attachment

### ISSUE THREAD

**Issue by rsrini83** [OfficeDev/ews-managed-api#142]
Hi,

Using OWA interface created a reference attachment mail, however when downloading the mail using EWS, however, HastAttachments flag is set to True and Attachments collections count is zero.  Used the below query to load the attributes,

PropertySet propSet = new PropertySet(BasePropertySet.FirstClassProperties, ItemSchema.HasAttachments, ItemSchema.NormalizedBody, ItemSchema.Attachments, ItemSchema.Categories, ItemSchema.InternetMessageHeaders, ItemSchema.AllowedResponseActions, ItemSchema.InReplyTo);

Please guide me how to download this. 

Btw,  in EWS response i didnt find any attachment tags which is captured via EWS trace.

**Comment by davster** [comment]
Turn on response tracing and paste in the response XML document that shows the missing attachments.

**Comment by rsrini83** [comment]
[EWS.log](https://github.com/OfficeDev/ews-managed-api/files/1535896/EWS.log)

Here is the log which contains all request, responses.

**Comment by davster** [comment]
If I am looking at the right responses, it looks like from the body of the email that these are reference attachments, right?

**Comment by rsrini83** [comment]
Yes, I added as reference attachment using OWA. However, I don't see Attachment collection in the response.

**Comment by rsrini83** [comment]
Am i doing right or Is there another way to download ReferenceAttachment ?

**Comment by rsrini83** [comment]
Hello David,

Any update on this issue?

**Comment by davster** [comment]
Try using <t:RequestServerVersion Version="Exchange2015" or greater - reference attachments are not supported for 2013 requests.
